tasks/main.yml

```diff
@@ -6,20 +6,29 @@
 # Examples of some tasks:
 - name: Ensure required packages are installed
```

Comment:
Remove tasks that ensure packages and services if there are no packages and services

Reply:
I think I should add a dependency on the lvm2 package.  I'll do that.